cli/command/container/run.go

```diff
@@ -104,6 +104,7 @@ func runRun(ctx context.Context, dockerCli command.Cli, flags *pflag.FlagSet, ro
 		}
 	}
 	copts.env = *opts.NewListOptsRef(&newEnv, nil)
+	copts.autoRemove = dockerCli.ConfigFile().AutoRemove
```

Comment:
This will always overwrite the `autoRemove` with the value from `config.json` (so `false` if it's not available). regardless of the passed CLI flags.

We should still allow the user to pass `--rm=false` to override the default `autoRemove: true`.

```suggestion
    if !flags.Changed("rm") {
   	copts.autoRemove = dockerCli.ConfigFile().AutoRemove
   }
```

Reply:
Ah! Good point. I always get a bit confused with Cobra options 🐍